dummy/controllers/menuController/menuController.js

```diff
@@ -0,0 +1,45 @@
+import menuDb from '../../db/menu';
+import fieldValidation from '../../validation/mealFieldRequiredValidation';
+
+// const { mealFieldRequiredValidation } = Validation;
+
+// eslint-disable-next-line require-jsdoc
+export default class Menu {
+  // eslint-disable-next-line valid-jsdoc
+  /**
+   *
+   * @param {*} req  THE REQUEST OBJECT
+   * @param {*} res   THE RESPONSE OBJECT
+   * @param {*} json  THE  RETURN JSON
+   */
+  static setMenu(req, res) {
+    const {
+      img, title, description, price
+    } = req.body;
+    fieldValidation(img, title, description, price, res);
```

Comment:
Hmm. What exactly is the purpose of having this method here as validation? The code beneath would still run whether or not validation fails.